### ISSUE TITLE
Add listings to marketplaces

### ISSUE THREAD

**Issue by lucadgtrip** [OriginProtocol/origin#2464]
We are doing several workshops and webinars to teach Hosts to create a listing in our marketplace.
It is proving very hard for many of them to deal with Metamask and even when we tell them to bring their laptop more than half of them show up with just a smartphone.
The experience on the Origin App is on the other hand incredibly straightforward and it makes me feel we are quite close to potential mass adoption, at least in terms of usability.
The only bump in the road is acquiring ETH for the gas but most of them somehow manage in sending us over their public address so we can send them some.
Unfortunately to have the listings on our marketplace creator requires to create the listing from the browser, which brings us back the problems above and the laptop.
If they could do everything on the smartphone it would be much easier to onboard them.

So, how do we bring the listing they created on our marketplace?
A few ideas
a) if they start from https://dapp.tripscommunity.com/ (or any other marketplace) and click on Get Started, they are brought to the Play/Apps store, download the app and the system knows the listing has to be added to Trips. That could also cover the referral campaigns when the code given is https://dapp.tripscommunity.com/#/welcome/366cdf61af2
b) a code: from the app, before or after adding the listing they are asked to insert a code (e.g. Trips) and this will attach their listing to the marketplace. The code could be manually created by the marketplace creator in order to avoid difficult to type ones.
c) at the end of the listing creation process on the dapp they are presented with a marketplaces list or a search box where they pick the marketplace they want to be added to. The list could filter the compatible marketplaces (if I add a t-shirt I won't see services marketplaces for instance).
d) marketplace admins can add manually listings they see on the Origin Dapp. A notification is sent to the listing owner who can then accept or refuse.

Once added, they go first in a waiting list and the marketplace admins will add them.
Not curated marketplaces could skip this step.

I think we should give users an interface where they see and manage the marketplaces in which they want their listings to appear.
It is a really powerful concept and **makes them understand how superior the open approach is** to today's situation where you have completely unrelated accounts, listing, calendars, reviews and so on.
When I explain this, they get it and many of them start to understand what we are talking about and why all this makes sense.

In the short term a) or d) would be enough, but they would allow make places to thrive.

**Comment by nick** [comment]
Thanks @lucadgtrip. In the next few weeks we'll be releasing an update that allows new listings and profiles to be created without the need for Eth by leveraging Meta Transactions.. which should improve the onboarding and listing process significantly.

Agree we should also allow more fine grained listing management for marketplaces by allowing marketplace owners to manage which listings appear and it what order. Thanks for you feedback 🙂

**Comment by lucadgtrip** [comment]
now that there's even the meta transactions there is no way we're sending people to https://dapp.tripscommunity.com to create their listing: **the Origin mobile App is the way to go.**
What's happening though is that listings are growing in https://www.shoporigin.com and non in https://dapp.tripscommunity.com.
That's fine, but it would be great if we could manually add to our marketplace the ones which fit the required criteria.
The way we envision it now is:
- Origin App: not curated
- Trips: curated (as per our White Paper)
- Other Dapps: they can ask Hosts to add the listings (curated or not, it's up to them).

So, basically the Origin App becomes the first, easy, step in this world.

What we'd need short term: an easy way for us to add listings from the Origin App to Trips.
Can this already be done manually if we ask you to?
That would allow us to start and not waste your time in developing a feature (whose specs will probably change in time anyway)
